dashboard/pages/screening/callbacks.py

```diff
@@ -215,11 +227,41 @@ def on_outlier_purge_stage_entry(
     )
 
 
+def on_outlier_purge_stage_entry_load_settings(
+    current_stage: int,
+    value: float,
+    saved_data: dict,
+) -> list[bool]:
+    """
+    Callback for the stage 2 entry
+    Change value of checkbox if json was loaded
+
+    :param current_stage: current stage index of the process
+    :param value: z threshold, slider value
+    :param saved_data: dict with loaded data
+    :return: value for checkbox
+    """
```

Comment:
to be fair - imo this setting does not really need saving for later - it is **purely visual** and does not impact the processing of the data, which the settings systems has to facilitate